src/ontology/hp-edit.owl

```diff
@@ -26952,23 +27086,33 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0000710> <http://purl.obolibrary.o
 
 # Class: <http://purl.obolibrary.org/obo/HP_0000711> (Restlessness)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:28332736") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0000711> "A state of unease characterized by diffuse motor activity or motion, subject to limited control, nonproductive, or disorganized behavior.")
```

Comment:
Is the loss of PMID acceptable?

Reply:
The PMID wasn't copied to the original spreadsheet where people were editing, but I don't think it is a good reference for this term. So yes, I would leave it out.